src/Protector.php

```diff
@@ -570,25 +577,36 @@ function () use ($request, $localDisk, $relativePath, $chunkSize, $sanctumIsActi
     /**
      * Returns the disk which is stated in the config. If no disk is stated the default filesystem disk will be returned.
      *
+     * @param string|null $diskName
      * @return FilesystemAdapter
      */
-    public function getDisk(): FilesystemAdapter
+    public function getDisk(?string $diskName = null): FilesystemAdapter
+    {
+        return Storage::disk($this->getDiskName($diskName));
+    }
+
+    /**
+     * @param string|null $diskName
+     * @return string
+     */
+    public function getDiskName(?string $diskName = null): string
```

Comment:
does it really make sense to give a diskname into this method just for it to return it

Reply:
Method was removed.